content/storage/postgres/docker-compose-for-postgres.md

```diff
@@ -88,24 +95,29 @@ COMPOSE_PROJECT_NAME=db-compose
 tree . -a
 .
 ├── .env
-├── docker-compose.yml
+├── docker-compose.[yml / yaml]
 └── init
     └── 1.create-user-table.sql
 ```
 
 #  Useful Docker compose commands
-- To launch a container.
+- To launch a container
 ```sh
 docker-compose up
 ```
 
-- To remove a container with its volumes.
+- To launch a container as a daemon
```

Comment:
I think -d is a detach right? No attach your containers in a terminal/session? I think it is okay during develop because you can see all logging message on a terminal immediately. 
https://docs.docker.com/compose/reference/up/

Reply:
I think so, I just added another choice for developers if they would like to run it as a daemon

Reply:
@dmakeroam, in Linux when you run a process in background, does it mean you run it as a daemon.
Is normal Docker engine runtime run as a daemon already?
To launch a process attach to terminal, doesn't matter that process to run a daemon right?
Please correct me because I concern if this comment can lead to misunderstanding.

Reply:
@dmakeroam after you have took action on this feedback, I will merge your PR.
Thanks.

Reply:
No, it cannot mean a daemon, if you don't put the command in systemd unit. Docker engine runtime doesn't be run as a daemon by default, you also need to apply "systemctl enable docker" or "set an option" in the Docker UI.

"To launch a process attach to terminal, doesn't matter that process to run a daemon right?" 

It doesn't matter if you'd like to throw your all logs to terminal.

*** By the way I'll update a word from daemon to background process to avoid misunderstand

Reply:
@dmakeroam  Thank you so much, now I learnt from you.